scripts/env-generate.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+# Ask for emails
+echo "Enter email addresses to filter commits by (press Enter when done):"
+EMAILS=""
+while true; do
+  read -p "Email: " email
+  if [[ -z "$email" ]]; then
+    break
+  fi
+  if [[ -z "$EMAILS" ]]; then
+    EMAILS="$email"
+  else
+    EMAILS="$EMAILS,$email"
+  fi
+done
+
+# Ask for repositories
+echo "Enter repository URLs (HTTPS or SSH) to monitor (press Enter when done):"
+REPOS=""
+while true; do
+  read -p "Repository URL: " repo
+  if [[ -z "$repo" ]]; then
+    break
+  fi
+  if [[ -z "$REPOS" ]]; then
+    REPOS="$repo"
+  else
+    REPOS="$REPOS,$repo"
+  fi
+done
+
+# Ask for monorepo URL
+read -p "Enter the monorepo remote repository URL: " MONOREPO_REPO
+
+# Ask for GitHub email
+read -p "Enter your GitHub email (from https://github.com/settings/emails): " GITHUB_EMAIL
+
+# Ask for GitHub email
+read -p "Enter your GitHub username: " GITHUB_USER
+
+# Generate .env file
+cat > ../.env <<EOL
```

Comment:
This creates the `.env` outside of the repository, should be at repo root
```suggestion
cat > .env <<EOL
```